config.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
for distribution?

we could add that to the bundle: https://github.com/opensearch-project/opensearch-build/blob/main/config/opensearch_dashboards-default.x.yml#L199

and then development we can avoid a specific starting up dev instructions since it would be a little bit confusing to clone this repo and it not to be available out of the box.

Reply:
Makes sense, thanks for clarification!